packages/dapp/src/components/CreatePresentationDisplay/index.tsx

```diff
@@ -43,13 +43,31 @@ const CreatePresentationDisplay = () => {
     }),
     shallow
   );
+  const { didMethod } = useMascaStore((state) => ({
+    didMethod: state.currDIDMethod,
+  }));
 
   const api = useMascaStore((state) => state.mascaApi);
 
   const [format, setFormat] = useState('JWT');
   const [advanced, setAdvanced] = useState(false);
   const [challenge, setChallenge] = useState('');
   const [domain, setDomain] = useState('');
+  const [availableProofFormats, setAvailableProofFormats] = useState([
+    'JWT',
+    'JSON-LD',
+    'EIP712Signature',
+  ]);
+
+  useEffect(() => {
+    if ((didMethod && didMethod === 'did:ethr') || didMethod === 'did:pkh') {
```

Comment:
This `if (didMethod === 'did:ethr' || didMethod === 'did:pkh') {` should probably be good enough ?